README.md

```diff
@@ -1,8 +1,24 @@
 # Booster Catalog
-Set of known example applications (Boosters) conforming to the minimal set of requirements necessary to be served by launch.openshift.io.
-
-- next: this is the branch from which we'll cut the next tag. Production does not use this branch, it is only used in the staging environment.
-- openshift-online-free; this branch is used in staging AND production. Only the Boosters capable of running in the OSO Free environment are included here.
+Set of known example applications (Boosters) conforming to the minimal set of requirements necessary to be served by developers.redhat.com/launch.
+
+Branching and tagging strategy:
+- `master`:
```

Comment:
What's the point of this `master` branch? For example, up until not long ago, `master` didn't mean anything for the Spring Boot boosters (all the work happened in the `sb-1.5.x` branch).

Reply:
This doesn't describe a `master` branch of the booster's repo :-) This is about `master` branch of the catalog.

Reply:
Yes, but you say a line below that it should point to the `master` branch of the boosters… which is why I pointed out its dubious (to me) utility :)

Reply:
Ah I see! How do I formulate it better? If I just say "the latest development branch of the boosters", would that be more readable for you?

Reply:
That would make sense. However, I still don't understand the purpose of this branch since it is not used by launcher, though that's not an issue with this updated documentation, more an issue with launcher branches… :)

Reply:
I also should add that we (Spring Boot team) haven't been updating that branch at all because we don't get why we need to do so. I think we could do away with having to maintain another branch for no obvious reason.

Reply:
Fixed (I hope :-) ).

Reply:
@metacosm You don't need to update the branch at all, it's always up to date for you. Currently, only Swarm needs to update this branch when doing a "productized" booster release, as Swarm doesn't use a separate branch (or repo) for "productized" boosters and releases them from the same branch as community.

The reason for this branch existence might be slightly dubious maybe, but you could use it to test the very very latest development of boosters (that aren't tagged/released yet) through Launcher.

Reply:
Thanks for the update.